### PR TITLE
gitignore: merge gitignore files

### PR DESCRIPTION
**Pull request by bergwolf**
Move cli/.gitignore and virtcontainers/.gitignore to top dir.

Fixes: #106

### CODE REVIEW

**Comment by laijs** [comment]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,19 @@
-/build-runv
+*.patch
+*.o
+*.swp
 /kata-runtime
 /data/kata-collect-data.sh
+/cli/coverage.html
+/cli/config-generated.go
+/cli/data/kata-collect-data.sh
+/cli/data/kata-collect-data.sh.in
```
Comment (jodh-intel) [inline .gitignore]: These two `kata-collect-data.sh` lines aren't needed as the script now lives in the top level `./data/` directory (see one of the lines above for the new location).

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+/cli/data/completions/bash/kata-runtime
+/cli/data/completions/bash/kata-runtime.in
```
Comment (jodh-intel) [inline .gitignore]: These two `completions` lines can be dropped for now (see #110 though).

**Comment by bergwolf** [comment]
@jodh-intel updated.

**Comment by jodh-intel** [comment]
lgtm

**Comment by devimc** [comment]
lgtm

**Pull request merged**